### DISCUSSION TITLE
Intellichlor Special Behavior

### DISCUSSION THREAD

**Discussion by mzuniga51** [tagyoureit/nodejs-poolController#500] [General]
@rstrouse Hi.  Remember that at some point you helped me to add a special behavior so that when I turn on the spa circuit the Intellichlor would go to 100%?   I lost that behavior when I went from having an additional Nixie controller operating my aux RS485 controller to directly incorporating the aux controller into my main instance.  Any way I can set that behavior again under the new setup?  Many thanks!

**Comment by tagyoureit** [comment]
Was that from this issue: https://github.com/rstrouse/nodejs-poolController-dashPanel/issues/52?  Can you re-upload a packet capture so we can see your latest config?

And just to clarify, you want the main pool's spa circuit to set the chlorinator to 100% on the 'remote' system?  And should it be off or some other % when the circuit is off?

**Comment by mzuniga51** [comment]
I'm talking about this one: https://github.com/rstrouse/nodejs-poolController-dashPanel/issues/53.  I lost this when I went back to a single njsPC instance.

And yes, this is exactly what I want.  100% is fine.  I think you did the first time using the heat on/off virtual circuit as the heat is always turned on when Spa is on.
[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/8964530/replay.zip)

**Reply by tagyoureit** [reply]
You should be able to use the same exact logic... just change the host to 'localhost' (or an ip) and double check the id's for the two chlorinators.  

```
{
  "context": {
    "name": "njsPCAux",
    "options": {
      "method": "POST",
      "path": "/",
      "headers": {
        "CONTENT-TYPE": "application/json"
      }
    },
    "vars": {}
  },
  "events": [
    {
      "name": "virtualCircuit",
      "description": "Set the superchlor when the spa heater turns on or off",
      "body": "@bind={id:1, isDosing: data.isOn};",
      "enabled": true,
      "options": {
        "host": "localhost",
        "port": 4200,
        "method": "PUT",
        "path": "/config/chlorinator"
      },
      "filter": "@bind=data.id === 245;"
    }
  ]

}
```

When the spa heater (245) changes, set the chlorinator dosing to the heater isOn value.  Give it a shot.

**Comment by mzuniga51** [comment]
Sorry I've been away for a few days.  So, this should go in the
controllers' config.json?  Under integrations like last time?  Thanks!
